README.rst

```diff
@@ -72,7 +72,7 @@ Building tapinstall (optional)
 The easiest way to build tapinstall is to clone the Microsoft driver samples
 and copy the source for devcon.exe into the tap-windows6 tree. Using PowerShell::
 
-  $ git clone https://github.com/Microsoft/Windows-driver-samples
+  $ git clone https://github.com/OpenVPN/Windows-driver-samples.git
```

Comment:
Our fork has a patch that ensures that devcon.exe is statically linked. I recall you mentioned it does not work (in some scenarios, at least). But we need accomplish static linking somehow. Or is this fixed in recent versions of MS Windows driver samples?

Otherwise this PR looks good afaics.